Makefile

```diff
@@ -140,8 +142,8 @@ test-all-valgrind: test-build
 	$(PYTHON) tools/test.py --mode=debug,release --valgrind
 
 test-ci: | build-addons
-	$(PYTHON) tools/test.py -p tap --logfile test.tap --mode=release \
-		addons message parallel sequential
+	$(PYTHON) tools/test.py $(TEST_CI_ARGS) -p tap --logfile test.tap --mode=release \
+    --flaky-tests=$(FLAKY_TESTS) addons message internet parallel sequential
```

Comment:
Indentation. Also shouldn't the default behaviour of flaky tests be don't care in CI?